raw_data/preprocessed/africa/kenya_english.txt

```diff
@@ -0,0 +1,471 @@
+
+Issue of emission licence
+(81) [1] Before issuing a licence in respect of emissions, the Authority shall: [a] consider the possible effects of the emissions on the quality of ambient air; [b] consider existing licences affecting the same air resource; [c] give due regard to the requirements for the residents, human settlements and other industrial and commercial activities; [d] solicit the comments of relevant Local Authorities and concerned organisations; [e] where the information accompanying the application appears inadequate, require the applicant to furnish further information relating to the undertaking in question, its location, materials, technology design or other appropriate matters; [f] where it appears necessary to conduct an environmental impact study, require the applicant to conduct an environmental impact assessment study in respect of the undertaking in question in accordance with the provisions of Part VI. [2] An emission licence issued under this Act shall be in a prescribed form, be subject to such conditions as may be prescribed or as may be specified in the licence and shall remain valid for such period and may be renewed for such periods as may be prescribed or specified in the licence. [3] Where the Authority rejects an application for the grant of an emission licence, it shall within twenty one days of its decision, notify the applicant in writing of its reasons for such refusal.
+
+Emissions by motor vehicles and other conveyances
+(82) No owner or operator of a motor-vehicle, train, ship, aircraft or other similar conveyance shall: [a] operate it in such a manner as to cause air pollution in contravention of the established emission standards; or [b] import any machinery, equipment, device or similar thing that will cause emissions into the ambient air in contravention of prescribed emission standards.
+
+Additional licensing procedures
+(83) The Authority may establish additional procedures for the application and grant of any licence under this Act and impose such conditions as it may deem appropriate.
+
+Cancellation of emission licence
+(84) The Authority may, in writing, cancel any emission licence: [a] if the holder of the licence contravenes any provisions of this Act or of any regulations made under it; [b] if the holder fails to comply with any condition specified in the licence; or [c] if the Authority considers it in the interest of the environment or in the public interest so to do.
+
+Register of emission of licences
```

Comment:
Should this read `Register of emission licenses`?

Reply:
Although the legislation states otherwise, I agree grammatically it should be edited and I have changed it accordingly.